src/utils.ts

```diff
@@ -0,0 +1,78 @@
+import * as core from '@actions/core'
+
+interface FieldValueType {
+  [key: string]: string
+}
+
+interface OptionType {
+  id: string
+  name: string
+}
+
+interface ProjectFieldsType {
+  options: OptionType[]
+  name: string
+  dataType: string
+  id: string
+}
+
+/**
+ * Returns an array of field updates for the GraphQL API
+ 
+ * @param fieldValues
+ * @param projectFields
+ * @returns {Promise<[]>}
+*/
+
+export async function getFieldUpdates(fieldValues: {[key: string]: string}, projectFields: []) {
+  const array: ProjectFieldsType[] = projectFields
+
+  const result = []
+
+  //remove empty objects from array of objects
+  const isNotEmpty = (objs: Record<string, any>) =>
+    !(objs && Object.keys(objs).length === 0 && objs.constructor === Object)
+  const clearedProjectFields = array.filter(isNotEmpty)
+
+  for (const [fieldName, fieldValue] of Object.entries(fieldValues)) {
+    const foundField = clearedProjectFields.find(
+      projectField => projectField.name.toLowerCase() === fieldName.toLowerCase(),
+    )
+
+    if (!foundField?.dataType) {
+      core.debug(`Could not find field with name ${fieldName}`)
+      continue
+    }
+    // TODO: Add support for other data types (e.g. ITERATIONS)
+    switch (foundField.dataType) {
+      case 'SINGLE_SELECT':
+        const optionFound = foundField.options.find(option => option.name.toLowerCase() === fieldValue.toLowerCase())
+
+        if (!optionFound) {
+          core.debug(`Could not find option for field ${foundField.name} with value ${fieldValue}`)
```

Comment:
```suggestion
          core.warning(`Could not find option for field ${foundField.name} with value ${fieldValue}`)
```

This should be a warning.